src/client/handlers/gestaltsGestaltList.ts

```diff
@@ -13,14 +13,17 @@ class GestaltsGestaltListHandler extends ServerHandler<
   ClientRPCRequestParams,
   ClientRPCResponseResult<GestaltMessage>
 > {
-  public async *handle(): AsyncGenerator<
-    ClientRPCResponseResult<GestaltMessage>
-  > {
+  public async *handle(
+    input_,
+    connectionInfo_,
```

Comment:
They're not being used here. I may have gotten the type of underscore mixed up.

Leading means unused and trailing means alternative/internal? I'll fix them up.

Reply:
So we have 3 naming conventions:

1. `this._x` means it is externally immutable, but internally mutable, this is usually combined with `get x(){ return this._x; }`.
2. `f(_a)` means un-used.
3. `a_` is alternative, because this is the most similar to `a'`, which is how haskell conventionally indicates an "updated" variable with the same meaning

Reply:
I wasn't aware that 1. was a convention we were using but looking at the code again I can see that this is the case. I'll fix the usage.

Reply:
This has been fixed.